### PR TITLE
fixed

### PR DESCRIPTION
**Pull request by shihyuntang**
hw done!!!!!

### CODE REVIEW

File: sytang/HW1.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import matplotlib.pyplot as plt
+
+
+# Fixing random state for reproducibility
+np.random.seed(19680801)
+
+# Compute areas and colors
+N = 150
+r = 2 * np.random.rand(N)
+theta = 5 * np.pi * np.random.rand(N)
```
Comment (cjtu) [inline sytang/HW1.py:14]: This should be 2

**Review by cjtu — COMMENTED** [review]
I left some comments

File: sytang/HW1.py (hunk 1 above)
Comment (shihyuntang) [reply sytang/HW1.py:14]: i see.....

**Comment by cjtu** [comment]
Nice work!!

**Pull request merged**